test/persistence-hooks.test.js

```diff
@@ -1,4 +1,5 @@
 var should = require('./init');
 var suite = require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
 
-suite(global.getDataSource(), should);
+suite(global.getDataSource(), should, 
+  {replaceOrCreateReportsNewInstance: true});
```

Comment:
In order to minimise changes in the future, please use the following style:

```js
suite(global.getDataSource(), should, {
  replaceOrCreateReportsNewInstance: true,
});
```

That way a new flag can be added by simply adding a new line, instead of having to touch multiple lines in your solution.